### ISSUE TITLE
Добавить очистку resize_cache 

### ISSUE THREAD

**Issue by kuntashov** [Mediahero/bitrix-clear-upload#3]
Неиспользованные файлы также накапливаются в resize_cache и их тоже желательно удалять.

**Comment by loadpm** [comment]
rm -rf resize_cache/*  решит проблему.

**Comment by mytraz** [comment]
> rm -rf resize_cache/* решит проблему.
> 
с таким успехом можно сразу вместо resize_cache писать upload, чего мелочиться

**Comment by kuntashov** [comment]
Нет, задачей подразумевалось именно удаление только тех файлов, которые раньше принадлежали удаленным элементам инфоблоков, а не слепое удаление.

К сожалению, для меня сейчас задача не актуальна, но с удовольствием приму PR, решающий ее :sunglasses:
